AdyenActions/Components/3DS2/Action handlers/ThreeDS2CoreActionHandler.swift

```diff
@@ -8,6 +8,13 @@ import Adyen
 import Adyen3DS2
 import Foundation
 
+// TODO: Remove this once 3ds2 sdk is released.
+extension NSError {
```

Comment:
Will this be removed?

Reply:
Yes, indeed will remove it once i create the 3ds2sdk release today/tomorrow. Thanks. Will close this comment once i do that.

Reply:
Released sdk version 2.3.2. So removed this code.